tests/test_compressor_fuzzing.py

```diff
@@ -767,7 +767,7 @@ def test_data_equivalence(self, original, threads, use_dict):
         dctx = zstd.ZstdDecompressor(**kwargs)
 
         for i, frame in enumerate(result):
-            self.assertEqual(dctx.decompress(frame), original[i])
+            self.assertEqual(dctx.decompress(frame.tobytes()), original[i])
```

Comment:
With the rust backend the test was erroring out becase `frame` wasn't a buffer but a `zstandard.backend_rust.BufferSegment` - not sure what the best way to fix it is...
This does work, so I added the conversion